static/panes/executor.js

```diff
@@ -39,9 +39,6 @@ var utils = require('../utils');
 var LibUtils = require('../lib-utils');
 var PaneRenaming = require('../widgets/pane-renaming').PaneRenaming;
 
-require('../modes/asm-mode');
-require('../modes/ptx-mode');
-
```

Comment:
Instead of scattering mode imports everywhere, they're all alphabetically sorted in _all